p/themes/base-theme/frss.css

```diff
@@ -37,6 +37,9 @@
 	--frss-scrollbar-track-hover: #0001;
 
 	--frss-loading-image: url("loader.gif");
+
+	font-size: 16px; /* do we really need it or should we trust the browser config? */
```

Comment:
I would definitely prefer to *respect* (not trust!) the browser config. If someone changes it they're doing so for a reason.

Reply:
Any suggestion how to change? delete this line? `font-size: 100%`?

Reply:
Just delete, yes.

Reply:
Would it makes sense to have an option to set/change the default size within FreshRSS?

Reply:
If you think it wouldn't make things too crowded under appearance I suppose. Is it really necessary though? It's really easy to adjust the zoom level and browsers remember it.

Reply:
Font size and zoom levels are 2 different things ;)

You could change the font-size to 20 and leave the zoom level on 100%.

I propose to have it right now without a config option. Will see if someone will open a feature request/bug issue report